torch_geometric/distributed/partition.py

```diff
@@ -267,17 +310,24 @@ def generate_partition(self):
                         'size': (data.num_nodes, data.num_nodes),
                     }, osp.join(path, 'graph.pt'))
 
-                torch.save(
-                    {
-                        'global_id': node_id,
-                        'feats': dict(x=part_data.x),
-                    }, osp.join(path, 'node_feats.pt'))
+                nfeat = {
+                    'global_id': node_id,
+                    'feats': dict(x=part_data.x),
+                }
+                if is_node_level_time:
+                    nfeat.update({'time': data.time})
+
+                torch.save(nfeat, osp.join(path, 'node_feats.pt'))
+
                 if 'edge_attr' in part_data:
-                    torch.save(
-                        {
-                            'global_id': edge_id,
-                            'feats': dict(edge_attr=part_data.edge_attr[perm]),
-                        }, osp.join(path, 'edge_feats.pt'))
+                    efeat = {
+                        'global_id': edge_id,
+                        'feats': dict(edge_attr=part_data.edge_attr[perm]),
+                    }
+                    if is_edge_level_time:
```

Comment:
Is it expected that we only add `edge_time` in case there exists edge features?

Reply:
While writing an e2e script that uses the MovieLens dataset (specifically: MovieLens(path, model_name='all-MiniLM-L6-v2') I noticed that there were no edge attributes there. So I had to move edge_time outside the condition. Does it make sense?

Reply:
Yes, but here it is inside the condition, right? Maybe I am misunderstanding.

Reply:
You are right. Here is the fix: [#8815](https://github.com/pyg-team/pytorch_geometric/pull/8815)